### ISSUE TITLE
dnsdbq 2.6.3 error parsing batch file $OPTIONS

### ISSUE THREAD

**Issue by eddocode** [dnsdb/dnsdbq#214]
I get `dnsdbq: warning: batch option parse error: superfluous non-arguments in $OPTIONS` message when I use dnsdbq version 2.6.3. However, when I use dnsdbq version 2.6.0, I do not get this error. Any help/suggestion would be greatly appreciated it!

Version 2.6.3
```
user@vm:~$ dnsdbq -v
dnsdbq: version 2.6.3
user@vm:~$ cat batch.txt 
$OPTIONS -A 2022-01-01 -B 2022-05-01
rdata/ip/198.35.26.96
user@vm:~$ dnsdbq -f < batch.txt > output.txt
dnsdbq: warning: batch option parse error: superfluous non-arguments in $OPTIONS
```

Version 2.6.0
```
user@vm:~$ dnsdbq -v
dnsdbq: version 2.6.0
user@vm:~$ cat batch.txt 
$OPTIONS -A 2022-01-01 -B 2022-05-01
rdata/ip/198.35.26.96
user@vm:~$ dnsdbq -f < batch.txt > output.txt
```

**Comment by djw1149** [comment]
Thanks for the bug report.  I pushed a fix and am awaiting a code review to merge it.

**Comment by djw1149** [comment]
A fix is available with tag v2.6.4.

Thanks Vixie for the merge.
